src/sentry/static/sentry/app/views/releases/pullRequestLink.jsx

```diff
@@ -10,34 +10,54 @@ class PullRequestLink extends React.Component {
     inline: PropTypes.bool,
   };
 
-  getUrl = () => {
-    if (this.props.repository.provider.id === 'github') {
+  get providerId() {
+    if (!this.props.repository.provider) {
+      return null;
+    }
+
+    let id = this.props.repository.provider.id;
+    if (id.indexOf(':') > -1) {
+      return id.split(':').pop();
+    }
+    return id;
+  }
+
+  get url() {
+    let providerId = this.providerId;
+    if (providerId === 'github') {
       return this.props.repository.url + '/pull/' + this.props.pullRequest.id;
     }
+    if (providerId === 'gitlab') {
+      return this.props.repository.url + '/merge_requests/' + this.props.pullRequest.id;
+    }
     return undefined;
-  };
+  }
 
   render() {
-    let url = this.getUrl();
+    let url = this.url;
     let displayId = `${this.props.repository.name} #${this.props.pullRequest.id}: ${this
       .props.pullRequest.title}`;
+    let providerId = this.providerId;
 
     return url ? (
       <a
         className={this.props.inline ? 'inline-commit' : 'btn btn-default btn-sm'}
         href={url}
         target="_blank"
       >
-        {this.props.repository.provider.id == 'github' && (
+        {providerId == 'github' && (
```

Comment:
This one is less clear cut since we're totally just doing display. But if we had a common naming scheme i.e. `icon-<integration_name>` the other two variables (`style` and `size`) are exactly the same and we can do less of the `if providerId == <integration_name>` do this.

instead it could like more like:
```js
<InlineSvg
   src=`icon-{{providerName}}`
   style={{verticalAlign: 'text-top'}}	           
   size="14px"	         
/>	      
```